includes/admin-bar-form-manager-new-form.php

```diff
@@ -173,6 +173,9 @@ public function output_default_field_name_script() {
 				var gfCreateField = window.CreateField;
 				window.CreateField = function( nextId, type, index ) {
 					var field = gfCreateField( nextId, type, index );
+					if ( type == 'submit' ) {
```

Comment:
When possible, use strict comparison (`===` vs `==`). In some cases, we actually want a loose comparison but this would be great situation to use strict.